server/JsDbg.Core/WebServer.cs

```diff
@@ -21,7 +21,7 @@
 using System.IO;
 
 namespace JsDbg.Core {
-    
+

```

Comment:
Are these newlines deliberate?

Reply:
My editor "helpfully" removed trailing whitespace, let me see if I can find a good way to fix that...